fugit2.nvim-scm-1.rockspec

```diff
@@ -0,0 +1,37 @@
+package	= 'fugit2.nvim'
```

Comment:
This rockspec is for local test installs and, optionally, for manually uploading an `scm` (DEV) version.